### PR TITLE
[jsk_apc2016_common] Add graspability of new gripper

### LINKED ISSUE #1354: Graspability Definition for APC2016 objects
Like #1225 
It isn't in my milestone for now, but what I should do in the near future.

### PR DESCRIPTION
**Pull request by pazeshun**
Closes #1354

### CODE REVIEW

**Comment by pazeshun** [comment]
This PR also fixes wrong graspability of old gripper about rolodex_jumbo_pencil_cup.

**Comment by knorth55** [comment]
add test
https://github.com/start-jsk/jsk_apc/blob/master/jsk_apc2016_common/python/jsk_apc2016_common/tests/__init__.py#L27

**Comment by pazeshun** [comment]
Added test

**Comment by wkentaro** [comment]
Please check graspability variable to be in [1, 4] for both grippers.

File: jsk_apc2016_common/data/object_data_2016.yaml [hunk 1]
```diff
@@ -1,234 +1,234 @@
...
     gripper2015: 4
-  #   gripper2016:
+    gripper2016: 3
   stock: 1
   weight: 6
 - name: rolodex_jumbo_pencil_cup
   graspability:
-    gripper2015: 1
-  #   gripper2016:
+    gripper2015: 4
```
Comment (knorth55) [inline jsk_apc2016_common/data/object_data_2016.yaml]: gripper2015 change should be in different commit.

**Comment by pazeshun** [comment]
To #1549
